backend/Dockerfile.db-rollback

```diff
@@ -1,4 +1,4 @@
-FROM gradle:6.9.1-jdk11-hotspot AS build
+FROM gradle:7.5.1-jdk11-hotspot AS build
```

Comment:
`gradle:7.5.1-jdk11-hotspot` doesnt exisit in docker hub
![image](https://user-images.githubusercontent.com/4952042/193874618-8d203483-68a9-4923-bca9-7dd5d32439fe.png)

Reply:
aha! Fixed it. The hotspot bit must have been why it updated to a different version previously.